tutorials/printing.adoc

```diff
@@ -1,83 +1,213 @@
+
+
+=== Úkol 3 (0 bodů, ale povinný)
+
+*Ukliďte po sobě své pracoviště.* Kdo po sobě neuklidí, nedostane žádné body.
+
+
+== Tisk krok za krokem
+
+Kapitola vás provede tiskem krok za krokem. 
+Většinu z těchto informací se dozvítě osobně na cvičení, nicméně pokud se dostanete do situace, kdy si nejste jistí, nebo jste instrukce ze cvičení přeslechli nebo z paměti vytěsnili, tento průvodce by vám měl být pomocníkem.
+
+
+=== Příprava tiskárny
+ 
+Před spuštěním tisku je potřeba dodržet několik kroků. 
+V případě, že tiskárnu nezkontrolujeme, můžeme skončit se špatným výtiskem, nebo v horším případě s poškozenou tiskárnou.
+
+. Vizuálně se ujistěte, že tiskárna není nijak zjevně poškozená (porovnejte např. s fotografií výše).
+. Očistěte tiskovou desku od nečistot či pozůstatcích předchozích výtisků.
+. Zkontrolujte a případně xref:#_zavedení-filamentu[zaveďte filament]. Ujistěte se, že se špule může při tisknu volně otáčet a nezasekne se.
+. TODO: Co ještě?
+
+
+==== Vyjmutí filamentu
+
+Abychom mohli vyjmout filament z tiskárny, musí xref:#_předehřátí-tiskárny[být zahřátá] na teplotu tání materiálu.
+Pokud byste tiskárnu nezahřáli, mohl by se filament v extruderu zlomit a způsobit problémy.
+
+Zahřejte tiskárnu na požadovanou teplotu a v rozhraní OctoPrint se přepněte na záložku xref:#_ruční-ovládání[ručního ovládání], a klikněte na tlačítko `Retract`.
+
+Po celou dobu vytahování mějte ruku poblíž struny, jelikož po jejím úplném vytažení má struna tendenci vystřelit a zamotat se. Vyjmutou strunu vložte do malého očka na špuli, aby se filament nezačal zamotávat. Buďte při manipulaci s filamentem opatrní. Špuli sundejte z držáku a filament odložte.
+
+
+==== Zavedení filamentu
+
+Pravidlo předehřáté tiskárny platí úplně stejně jako u vytahování filamentu.
+
+image::../images/filamentguide.svg[Jak má filament vypadat, 300, float="right"]
+
+Zkontrolujte zakončení filamentu na závady. Filament může být:
+
+. Nerovný či polámaný
+. Mít roztavené zakončení
+. Být "nahlodaný" od ozubení extruderu
+
+V každém případě takovou část uřízněte. 
+
+Zařízněte filament do špičky a vložte jej do otvoru na extruderu. Netlačte filament do tiskárny násilím, stačí jen vložit a jemně přitlačit a v OctoPrintu na záložce xref:#_ruční-ovládání[ručního ovládání] klikněte na tlačítko `Extrude`.
+
+Ve chvíli, kdy tiskárna uchopí filament můžete jej pustit. Extrudujte filament tak dlouho, dokud nezačne z trysky vytékat plast. Dále extrudujte plast tak dlouho, dokud nezačne z trysky vytékat plast správné barvy (v trysce mohly být pozůstatky filamentu jiné barvy).
+
+
+==== ABS Juice
+
+K vylepšení přilnavosti tiskové desky se používají různé materiály - lepidla, ABS Juice, laky. 
+V učebně je k dispozici ABS Juice, používejte, prosím, na tiskárny v učebně pouze ten. 
+Pokud tisknete předmět, který by se mohl v průběhu tisku odlepit, naaplikujte na desku jemnou vrstvu ABS Juice.
```

Comment:
+ Tlusté nánosy ABS Juicu jsou nejen nechutné, ale také nežádoucí.

+ ABS Juice je malé množství ABS rozpuštěné v acetonu. Aceton je nebezpečná látka, dbejte zvýšené opatrnosti. V případě kontaktu s nekrytou částí těla okamžitě kontaktujte cvičícího. Při manipulaci zkontrolujte, že je otevřené okno. Ihned po vyjmutí štětce nádobu zavřete. Ponechání otevřené nádoby s acetonem bez dozoru je vážné porušení BOZP.